server/bleep/src/indexes/file.rs

```diff
@@ -42,10 +39,11 @@ use crate::{
 struct Workload<'a> {
     cache: &'a FileCacheSnapshot<'a>,
     file_filter: &'a FileFilter,
+    repo_ref: &'a RepoRef,
     repo_disk_path: &'a Path,
     repo_name: &'a str,
     repo_metadata: &'a RepoMetadata,
-    repo_ref: String,
+    repo_ref_str: String,
```

Comment:
Confusing having both `repo_ref` and `repo_ref_str` here. Why don't we just have `repo_ref` and transform into a `String` when needed.